pkg/resources/secret_with_basic_authentication_acceptance_test.go

```diff
@@ -88,20 +136,61 @@ func TestAcc_SecretWithBasicAuthentication_BasicFlow(t *testing.T) {
 				Config: config.FromModel(t, secretModelWithoutComment),
 				ConfigPlanChecks: resource.ConfigPlanChecks{
 					PreApply: []plancheck.PlanCheck{
-						planchecks.ExpectChange(secretModel.ResourceReference(), "comment", tfjson.ActionUpdate, sdk.String(comment), nil),
+						plancheck.ExpectResourceAction(secretName, plancheck.ResourceActionUpdate),
+						planchecks.ExpectChange(secretName, "comment", tfjson.ActionUpdate, sdk.String(comment), nil),
 					},
 				},
 				Check: assert.AssertThat(t,
-					resourceassert.SecretWithClientCredentialsResource(t, secretModelWithoutComment.ResourceReference()).
+					resourceassert.SecretWithClientCredentialsResource(t, secretName).
 						HasCommentString(""),
 				),
 			},
-			// destroy
+			// import with no fields set
 			{
-				Config:  config.FromModel(t, secretModel),
-				Destroy: true,
+				ResourceName:            secretName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"password"},
+				ImportStateCheck: importchecks.ComposeImportStateCheck(
+					importchecks.TestCheckResourceAttrInstanceState(helpers.EncodeResourceIdentifier(id), "name", id.Name()),
+					importchecks.TestCheckResourceAttrInstanceState(helpers.EncodeResourceIdentifier(id), "database", id.DatabaseId().Name()),
+					importchecks.TestCheckResourceAttrInstanceState(helpers.EncodeResourceIdentifier(id), "schema", id.SchemaId().Name()),
+					importchecks.TestCheckResourceAttrInstanceState(helpers.EncodeResourceIdentifier(id), "username", "bar"),
+					importchecks.TestCheckResourceAttrInstanceState(helpers.EncodeResourceIdentifier(id), "comment", ""),
+				),
 			},
 			// create with empty username and password
+			{
+				Config: config.FromModel(t, secretModelEmptyCredentials),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
It tests setting with empty username and password for an existing secret and the test below tests if it is possible to create such secret with empty credentials. But the naming is off since before, there was `destroy` before this step